psdk_interfaces/msg/MediaInfo.msg

```diff
@@ -0,0 +1,2 @@
+
+MediaFileInfo[]  file_data
```

Comment:
and here I also think it would be clearer if instead of `file_data` you would only have `files`. 
Then when you iterate over this vector it is a bit clearer that it is indeed a vector. You would have something like `media_file_info_msg.files[0].file_type`